utilities/similarity/requirements.txt

```diff
@@ -0,0 +1,6 @@
+boto3
```

Comment:
Remove this file and use instead `requirements.txt` that is at the root of this project (see other Dockerfiles).